src/Symfony/Component/DependencyInjection/Loader/FileLoader.php

```diff
@@ -14,6 +14,9 @@
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\Config\Loader\FileLoader as BaseFileLoader;
 use Symfony\Component\Config\FileLocatorInterface;
+use Symfony\Component\Config\Exception\FileLoaderLoadException;
```

Comment:
should be moved before FileLoader (we add new entries in alpha order, but keep existing ones asis)